Description: Remove stage1 extra dependencies (cargo 0.17)
 Fetching extra dev-dependencies is not yet supported by bootstrap.py
 and they are not need to build. However, cargo will try to download
 them before building stage1.
 .
 For patches for earlier versions of cargo, see the git history for this file.
From: Luca Bruno <lucab@debian.org>
From: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -58,12 +58,12 @@
 [target.'cfg(target_os = "macos")'.dependencies]
 core-foundation = { version = "0.4.4", features = ["mac_os_10_7_support"] }
 
-[target.'cfg(windows)'.dependencies]
-advapi32-sys = "0.2"
-kernel32-sys = "0.2"
-miow = "0.2"
-psapi-sys = "0.1"
-winapi = "0.2"
+#[target.'cfg(windows)'.dependencies]
+#advapi32-sys = "0.2"
+#kernel32-sys = "0.2"
+#miow = "0.2"
+#psapi-sys = "0.1"
+#winapi = "0.2"
 
 [dev-dependencies]
 bufstream = "0.1"
